assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * DashboardAllTrafficWidget component
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Widgets from 'googlesitekit-widgets';
+import { STORE_NAME as MODULE_ANALYTICS, FORM_ALL_TRAFFIC_WIDGET } from '../../../datastore/constants';
```

Comment:
Naming tweak for consistency 🙂 

```suggestion
import { STORE_NAME as MODULES_ANALYTICS, FORM_ALL_TRAFFIC_WIDGET } from '../../../datastore/constants';
```